docs/operating-scylla/procedures/config-change/rolling-restart.rst

```diff
@@ -23,6 +23,9 @@ Procedure
 
 .. include:: /rst_include/scylla-commands-start-index.rst
 
-5. Verify the node is up and has returned to the Scylla cluster using :doc:`nodetool status </operating-scylla/nodetool-commands/status/>`.
+5. Verify the node is up and has returned to the Scylla cluster, and CQL port is available
+
+   * :doc:`nodetool status </operating-scylla/nodetool-commands/status/>`.
+   * ``cqlsh -e "SHOW VERSION" || exit 1``
```

Comment:
This is just one option and it's not very practical.
A more practical option is to simply probe a corresponding CQL TCP port.

Reply:
> A more practical option is to simply probe a corresponding CQL TCP port.

why is it simpler?  users are familiar with CQLsh

Reply:
First, because there are many more people that are familiar with TCP than with `cqlsh`.
Second, probing a TCP port never requires authorization unlike using `cqlsh`.